irodsinfo/views_api.py

```diff
@@ -0,0 +1,49 @@
+"""REST API views for the irodsinfo app"""
+
+import logging
+
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+# Projectroles dependency
+from projectroles.plugins import get_backend_api
+
+from irodsinfo.views import IrodsConfigMixin
+
+
+logger = logging.getLogger(__name__)
+
+
+class IrodsConfigRetrieveAPIView(IrodsConfigMixin, APIView):
+
+    """
+    Retrieve iRODS configuration file for the current user.
+
+    **URL:** ``/samplesheets/api/investigation/retrieve/{Project.sodar_uuid}``
+
+    **Methods:** ``GET``
+
+    **Returns:**
+
+    - ``irods_environment``: Return user iRODS config (``json``)
```

Comment:
This should say `iRODS client environment`. JSON is a given, all responses are in JSON unless otherwise specified.